demosplan/DemosPlanCoreBundle/Command/Db/MigrateCommand.php

```diff
@@ -49,17 +49,32 @@ public function execute(InputInterface $input, OutputInterface $output): int
             $db = '--conn='.$db;
         }
 
+
+        $commands = [
+            "dplan:migrations:cache --env={$env}",
+            "doctrine:migrations:sync-metadata-storage {$db} --env={$env}",
+        ];
+
+        $migrationsConfigurationPath = DemosPlanPath::getProjectPath('app/config/project_migrations.yml');
+        $migrationsSyncCommand = 'doctrine:migrations:sync-metadata-storage --configuration ';
+        $migrationsCommand = 'doctrine:migrations:migrate --configuration ';
+        $lastRollupMigration = 'app/Resources/DemosPlanCoreBundle/DoctrineMigrations/2022/09/Version20220914133419.php';
```

Comment:
this would make more sense as a constant for me.

Reply:
Even if this is only used in this single context? Hm, I would change it, I do not really care, but feels a bit too much to me

Reply:
It's just a matter of code style. I don't really care either way.